GridSample-9 specification (#11703)
openvinotoolkit/openvino: docs/ops/image/GridSample_1.md

  * **Range of values**:
    * `false` - map extrema values to the center of pixels
    * `true` - map extrema values to the borders of pixels
  * **Type**: `boolean`
  * **Default value**: false
  * **Required**: *no*

* *mode*

  * **Description**: specifies the interpolation type used to calculate the output elements
  * **Range of values**: one of: `bilinear`, `bicubic` or `nearest`
  * **Type**: `string`
  * **Default value**: bilinear
  * **Required**: *no*

* *padding_mode*

  * **Description**: controls the handling of out-of-bounds coordinates. The denormalized coordinates might fall outside of the input tensor's area(values outside the grid).
  * **Range of values**: 
    * `zeros` - consider values in the padding to be zeros
    * `border` - the operator is supposed to select the nearest in-bounds pixel
    * `reflection` - repeatedly reflect the out-of bounds value until it points to a pixel that belongs to the image
  * **Type**: `string`
  * **Default value**: zeros
  * **Required**: *no*

**Inputs**

> *   **1**: `data` - Input tensor with data to be sampled. Any supported OV data type can be used. This input is expected to be a 4-dimensional tensor with NCHW layout. **Required.**

Reviewer: Please add *Type* section and use _T_ to describe types. You can take [Interpolate-4](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/image/Interpolate_4.md) spec as reference.

> *   **2**: `grid` - A 4-dimensional tensor containing floating point values of normalized sampling coordinates(pairs of floats). The shape of this tensor is [N, H_out, W_out, 2] **Required.**

Reviewer: Please list another type in the Type section to describe `grid` input type.

**Outputs**

> *   **1**: A 4-dimensional tensor with [N, C, H_out, W_out] shape. This tensor's data type is equal to the data type of the `data` input tensor. It contains the interpolated values calculated by this operator.

Reviewer: When you add _Type_ section, you can reuse type _T_ to mark type of input/output.

Reviewer 2: ```suggestion
*   **1**: A 4-dimensional tensor with [N, C, H_out, W_out] shape. This tensor's data type is same as the data type of the `data` input tensor. It contains the interpolated values calculated by this operator.
```

> # GridSample {#openvino_docs_ops_image_GridSample_9}

Reviewer 2: File name should change as well, shouldn't it?
